### DISCUSSION TITLE
Where to submit a bug bounty report?

### DISCUSSION THREAD

**Discussion by manunio** [neo-project/neo#3445] [Q&A] [answered]
I tried sending a report to the email mentioned at https://neo.org/bounty didn't receive any response. Saw that you have a program at https://immunefi.com/bug-bounty/neo so sent a report there. Just wanted to confirm which channel is active.

**Comment by steven1227** [comment, marked as answer]
@manunio hi, Thanks for your report. Immunefi is work, let us take a look.
